packages/keyring-eth-hd/package.json

```diff
@@ -61,7 +61,8 @@
     "@ts-bridge/cli": "^0.6.1",
     "@types/jest": "^29.5.12",
     "deepmerge": "^4.2.2",
-    "jest": "^29.5.0"
+    "jest": "^29.5.0",
+    "typescript": "~5.6.3"
```

Comment:
Not needed as of now, but since this package has already been "future-proof" for `ts-bridge` we might wanna add `typescript` as well.